code/modules/clothing/spacesuits/alien_suits.dm

```diff
@@ -143,23 +143,22 @@
 	desc = "An almost organic looking nonhuman pressure suit."
 
 /obj/item/clothing/under/vox
-	has_sensor = FALSE
 	icon = 'icons/obj/clothing/species/vox/uniforms.dmi'
 	species_restricted = list("Vox")
 	sprite_sheets = list(
 		"Vox" = 'icons/mob/clothing/species/vox/under/misc.dmi'
 		)
 
 /obj/item/clothing/under/vox/vox_casual
-	name = "alien clothing"
+	name = "Vox Jumpsuit"
```

Comment:
```suggestion
	name = "Vox jumpsuit"
```
Jumpsuit should be lower case here so when examined the sentence will look proper.